File: 백준/Bronze/10809. 알파벳 찾기/알파벳 찾기.cc
#include <stdio.h>
#include <string.h>

int main()
{
    char word[101];
    int index[26];    
    scanf("%s", word);
    
    for(int i = 0; i < 26; i++)
    {
        index[i] = -1;
    }
    for(int i = 0; word[i] != '\0'; i++)
    {
        int ch = word[i] - 'a';
        if(index[ch] == -1)
        {
            index[ch] = i;
        }
    }
    for(int i = 0; i < 26; i++)
    {
        printf("%d ", index[i]);
    }
    return 0;
}

File: 백준/Bronze/10809. 알파벳 찾기/알파벳 찾기.java
import java.util.Scanner;

public class Main
{
    public static void main(String[] args)
    {
        Scanner scanner = new Scanner(System.in);
        int[] index = new int[26];
        String word = scanner.next();
        
        for(int i = 0; i < 26; i++)
        {
            index[i] = -1;
        }
        
        for(int i = 0; i < word.length(); i++)
        {
            int ch = word.charAt(i) - 'a';
            if(index[ch] == -1)
            {
                index[ch] = i;
            }
        }
        
        for(int i = 0; i < 26; i++)
        {
            System.out.print(index[i] + " ");
        }
        scanner.close();
    }
}